src/AbstractPolytope.jl

```diff
@@ -128,6 +129,46 @@ end
     end
 end
 
+"""
+    minkowski_sum(P1::AbstractPolytope{N}, P2::AbstractPolytope{N};
+                  [apply_convex_hull]=true,
+                  [backend]=default_polyhedra_backend(P1, N)) where {N<:Real}
+
+Compute the Minkowski sum between two polytopes using their vertex representation.
+
+### Input
+
+- `P1`                -- polytope
+- `P2`                -- another polytope
+- `apply_convex_hull` -- (optional, default: `true`) if `true`, post-process the
+                         pairwise sums using a convex hull algorithm 
+- `backend`           -- (optional, default: `default_polyhedra_backend(P1, N)`)
+                         the backend for polyhedral computations used to
+                         post-process with a convex hull
+
+### Output
+
+A new polytope in vertex representation whose vertices are the convex hull of
+the sum of all possible sums of vertices of `P1` and `P2`.
+"""
+function minkowski_sum(P1::AbstractPolytope{N}, P2::AbstractPolytope{N};
+                       apply_convex_hull::Bool=true,
+                       backend=default_polyhedra_backend(P1, N)) where {N<:Real}
+    vlist1, vlist2 = vertices_list(P1), vertices_list(P2)
+    n, m = length(vlist1), length(vlist2)
+    Vout = Vector{Vector{N}}()
```

Comment:
Note that the code does not work if `n == 0` or `m == 0`.

Reply:
With the assertion, say if `n = 0` and `m > 0`, now it gives an error that the dimensions do not match.